packages/test/test-end-to-end-tests/src/test/summaries.spec.ts

```diff
@@ -505,6 +515,47 @@ describeCompat("Summaries", "NoCompat", (getTestObjectProvider, apis) => {
 		assert.strictEqual(sharedString1.getLength(), 203);
 		assert.strictEqual(sharedString2.getLength(), 203);
 	});
+
+	itExpects(
+		"attempt last summary if parent container disconnects",
+		[
+			{
+				eventName: "fluid:telemetry:Summarizer:Running:Summarize_Op",
+				summarizeReason: "lastSummary",
+			},
+		],
+		async function () {
+			const container = await createContainer(provider, {});
+			const { summarizer } = await createSummarizer(provider, container, {
+				runtimeOptions: {
+					summaryOptions: {
+						summaryConfigOverrides: {
+							state: "enabled",
+							minOpsForLastSummaryAttempt: 1, // This is the key setting
+							maxTime: 60 * 1000,
+							maxOps: 1000,
+							minIdleTime: 60 * 1000,
+							maxIdleTime: 60 * 1000,
+							runtimeOpWeight: 1,
+							nonRuntimeOpWeight: 1,
```

Comment:
This test is not sending any ops but a last summary is still happening. I assume this is because of system ops. Validating summaries with user ops is more interesting.
You should send an op and maybe change the `nonRuntimeOpWeight` to a high number to ensure that the last summary is triggered by user ops.

Alternatively, you could also add another test for last summary triggered by user ops and keep this for system ops.